### PR TITLE
[zh] Update text from ist0001 to ist0118

### PR DESCRIPTION
**Pull request by windsonsea**
- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/reference/config/analysis/ist0101/index.md [hunk 1]
```diff
@@ -5,15 +5,16 @@ owner: istio/wg-user-experience-maintainers
 test: no
 ---
 
-当一个 Istio 资源引用另一个不存在的资源时，会触发此消息。当 Istio 试图查找引用的资源，但是找不到它时，这将导致错误。
+当 Istio 资源引用另一个不存在的资源时，会触发此消息。
```
Comment (hanxiaop) [inline content/zh/docs/reference/config/analysis/ist0101/index.md:8]: 触发和出现可以统一一下，其他地方好像都是出现

File: content/zh/docs/reference/config/analysis/ist0101/index.md [hunk 2]
```diff
@@ -5,15 +5,16 @@ owner: istio/wg-user-experience-maintainers
...
+当 Istio 尝试查找引用的资源但找不到时，将导致此错误。
```
Comment (hanxiaop) [inline content/zh/docs/reference/config/analysis/ist0101/index.md:9]:
```suggestion
这会导致 Istio 尝试查找引用的资源但找不到这类的错误。
```

File: content/zh/docs/reference/config/analysis/ist0102/index.md [hunk 3]
```diff
@@ -5,7 +5,9 @@ owner: istio/wg-user-experience-maintainers
 test: no
 ---
 
-当命名空间缺少[注解](/zh/docs/reference/config/annotations/)来标记命名空间是否自动注入时会出现此错误，例如 `sidecar.istio.io/inject` 。
+当命名空间缺少 `sidecar.istio.io/inject`
```
Comment (hanxiaop) [inline content/zh/docs/reference/config/analysis/ist0102/index.md:8]: 这个有问题吧，命名空间只会有 istio-injection: enabled 标签。而且 annotation 现在已经不用了，用的都是 label。

File: content/zh/docs/reference/config/analysis/ist0103/index.md [hunk 4]
```diff
@@ -5,13 +5,13 @@ owner: istio/wg-user-experience-maintainers
...
 ---
 
-当 sidecar 不存在或不能正常工作时，将触发此消息。
+当 Sidecar 不存在或不能正常工作时，将触发此消息。
 
-最常见的情况是，您开启了自动注入，但之后没有重新启动您的 pod，导致 sidecar 丢失。
+最常见的情况是，您开启了自动注入，但之后没有重新启动您的 Pod，导致 Sidecar 丢失。
 
-为了解决这个问题，重新启动您的 pod，然后重试一次。
+为了解决这个问题，请重新启动您的 Pod，然后重试。
```
Comment (hanxiaop) [inline content/zh/docs/reference/config/analysis/ist0103/index.md:12]:
```suggestion
为了解决这个问题，请重新启动您的 Pod 以便进行注入。
```

File: content/zh/docs/reference/config/analysis/ist0103/index.md [hunk 5]
```diff
@@ -5,13 +5,13 @@ owner: istio/wg-user-experience-maintainers
 test: no
 ---
 
-当 sidecar 不存在或不能正常工作时，将触发此消息。
+当 Sidecar 不存在或不能正常工作时，将触发此消息。
 
-最常见的情况是，您开启了自动注入，但之后没有重新启动您的 pod，导致 sidecar 丢失。
+最常见的情况是，您开启了自动注入，但之后没有重新启动您的 Pod，导致 Sidecar 丢失。
```
Comment (hanxiaop) [inline content/zh/docs/reference/config/analysis/ist0103/index.md:10]:
```suggestion
最常见的情况是，您开启了自动注入，但之后没有重新启动您的 Pod，导致还未被注入 Sidecar。
```

File: content/zh/docs/reference/config/analysis/ist0109/index.md [hunk 6]
```diff
@@ -5,22 +5,24 @@ owner: istio/wg-user-experience-maintainers
 test: no
 ---
 
-当 Istio 检测到因[Virtual Service](/zh/docs/reference/config/networking/virtual-service)资源重复而导致冲突时，会出现该信息。比如，多个 Virtual Service 使用相同的主机名且连接 Gateway，会出现错误信息。需要注意的是，Istio 支持 Virtual Service 合并来连接入口网关。
+当 Istio 检测到因 [Virtual Service](/zh/docs/reference/config/networking/virtual-service)
+资源重复而导致冲突时，会出现此消息。比如，多个 Virtual Service 使用相同的主机名且连接网格 Gateway 时，
```
Comment (hanxiaop) [inline content/zh/docs/reference/config/analysis/ist0109/index.md:9]:
```suggestion
资源重复而导致冲突时，会出现此消息。比如，多个 VirtualService 使用相同的主机名且连接网格 Gateway 时，
```

File: content/zh/docs/reference/config/analysis/ist0109/index.md [hunk 7]
```diff
@@ -5,22 +5,24 @@ owner: istio/wg-user-experience-maintainers
...
+会出现一条错误消息。需要注意的是，Istio 支持合并挂接到入口网关的 Virtual Service。
```
Comment (hanxiaop) [inline content/zh/docs/reference/config/analysis/ist0109/index.md:10]:
```suggestion
会出现一条错误消息。需要注意的是，Istio 支持合并挂接到入口网关的 VirtualService。
```

File: content/zh/docs/reference/config/analysis/ist0109/index.md [hunk 8]
```diff
@@ -5,22 +5,24 @@ owner: istio/wg-user-experience-maintainers
...
 
 * 将冲突的 Virtual Service 合并为一个
 * 连接 Gateway 的 Virtual Service 使用唯一的主机名
 * 通过设置 `exportTo` 字段，将资源范围限定到指定的命名空间。
 
-## 示例{#examples}
+## 示例 {#examples}
 
-命名空间 `team1` 的虚拟服务 `productpage` 与命名空间 `team2` 的 Virtual Service `custom` 存在冲突的原因如下：
+命名空间 `team1` 的虚拟服务 `productpage` 与命名空间 `team2` 的虚拟服务 `custom` 存在冲突的原因如下：
```
Comment (hanxiaop) [inline content/zh/docs/reference/config/analysis/ist0109/index.md:22]: 和上面统一成 VirtualService 或者虚拟服务。

File: content/zh/docs/reference/config/analysis/ist0118/index.md [hunk 9]
```diff
@@ -33,13 +34,15 @@ spec:
...
 
-- 如果您知道 service 端口的协议，将端口重命名为 `<protocol>[-<suffix>]` 格式就行；
-- 如果您不知道 service 端口的协议，您需要[从 Prometheus 查询指标](/zh/docs/tasks/observability/metrics/querying-metrics/)
-    - 查询 `istio_requests_total{reporter="destination",destination_service_name="SERVICE_NAME",response_code="200"}[TIME_RANGE]`。如果您使用 Mixer v2 版本，
-      也可以查询 `istio_requests_total{reporter="destination",destination_service_name="SERVICE_NAME",response_code="200",destination_port="TARGET_PORT"}[TIME_RANGE]`。
-    - 如果有输出，您可以从记录中找到 `request_protocol`。例如，如果 `request_protocol` 是 `http`，则将端口重命名为 `http-foo`；
+- 如果您知道 Service 端口的协议，将端口重命名为 `<protocol>[-<suffix>]` 格式就行；
+- 如果您不知道 Service 端口的协议，您需要[从 Prometheus 查询指标](/zh/docs/tasks/observability/metrics/querying-metrics/)
+    - 查询 `istio_requests_total{reporter="destination",destination_service_name="SERVICE_NAME",response_code="200"}[TIME_RANGE]`。
+      如果您使用 Mixer v2 版本，也可以查询
```
Comment (hanxiaop) [inline content/zh/docs/reference/config/analysis/ist0118/index.md:44]: Mixer 可能可以改成 Telemetry

**Review by hanxiaop — COMMENTED** [review]
Do we have markdown translations that exist only in the ZH folders? I think we deleted some of the messages from the EN folders before.

**Comment by windsonsea** [comment]
Checked all are synced. The number of `ist*.md` in the zh folder is 45, same as that in the en folder.

File: content/zh/docs/reference/config/analysis/ist0101/index.md (hunk 1 above)
Comment (windsonsea) [reply content/zh/docs/reference/config/analysis/ist0101/index.md:8]: ok, `出现`看起来更通用一点

**Comment by hanxiaop** [comment]
> Checked all are synced. The number of `ist*.md` in the zh folder is 45, same as that in the en folder.

Great, thanks!

File: content/zh/docs/reference/config/analysis/ist0102/index.md (hunk 3 above)
Comment (windsonsea) [reply content/zh/docs/reference/config/analysis/ist0102/index.md:8]: Seems it's time to update the [en version of ist0102](https://github.com/istio/istio.io/blob/master/content/en/docs/reference/config/analysis/ist0102/index.md).

File: content/zh/docs/reference/config/analysis/ist0118/index.md (hunk 9 above)
Comment (windsonsea) [reply content/zh/docs/reference/config/analysis/ist0118/index.md:44]: 这个等 en 更新后再同步怎么样，记得 Mixer 的确被弃用了

File: content/zh/docs/reference/config/analysis/ist0102/index.md (hunk 3 above)
Comment (hanxiaop) [reply content/zh/docs/reference/config/analysis/ist0102/index.md:8]: Yes I think it needs to be updated. This message only occurs when we have no `istio-injection`, `istio.io/rev` label on the namespace, or we do not enable `.values.sidecarInjectorWebhook.enableNamespacesByDefault`. It's not related to any annotation at all.

File: content/zh/docs/reference/config/analysis/ist0109/index.md [hunk 10]
```diff
@@ -5,22 +5,24 @@ owner: istio/wg-user-experience-maintainers
 test: no
 ---
 
-当 Istio 检测到因[Virtual Service](/zh/docs/reference/config/networking/virtual-service)资源重复而导致冲突时，会出现该信息。比如，多个 Virtual Service 使用相同的主机名且连接 Gateway，会出现错误信息。需要注意的是，Istio 支持 Virtual Service 合并来连接入口网关。
+当 Istio 检测到因 [Virtual Service](/zh/docs/reference/config/networking/virtual-service)
```
Comment (hanxiaop) [inline content/zh/docs/reference/config/analysis/ist0109/index.md:8]:
```suggestion
当 Istio 检测到因 [VirtualService](/zh/docs/reference/config/networking/virtual-service)
```

File: content/zh/docs/reference/config/analysis/ist0102/index.md (hunk 3 above)
Comment (windsonsea) [reply content/zh/docs/reference/config/analysis/ist0102/index.md:8]: Ok, I will include this message into a new and separate PR.

**Review by hanxiaop — APPROVED** [review]
one small nit

**Review by wilsonwu — APPROVED** [review]
LGTM, thanks!

**Pull request merged**